frontend/src/components/account/EditAccountPage.js

```diff
@@ -469,6 +505,51 @@ export default function EditAccountPage({
             />
           </div>
         );
+      } else if (i.type === "selectwithtext") {
+        const valuesThatDisableField = [1, 3]; // we can manually set which types we don't want to have this feature for
+        const truthValues = valuesThatDisableField.map((val) => editedAccount.types.includes(val));
+        const shouldTextFieldBeDisabled =
+          truthValues.includes(true) || editedAccount.types.length === 0;
+
+        return (
+          <>
+            <div key={key} className={classes.infoElement}>
+              <SelectField
+                className={classes.selectOption}
+                options={i.options}
+                label={i.organization_size.name}
+                defaultValue={{
+                  name: i.options[i.value.organization_size]?.name,
+                  key: i.options[i.value.organization_size]?.name,
+                }}
+                onChange={handleChange}
+              />
+            </div>
+
+            <Typography className={classes.subtitle}>
+              {i.get_involved.name}
+              {i.get_involved.helptext && (
+                <Tooltip title={i.get_involved.helptext}>
+                  <IconButton>
+                    <HelpOutlineIcon className={classes.helpIcon} />
+                  </IconButton>
+                </Tooltip>
+              )}
+            </Typography>
+            <TextField
+              disabled={shouldTextFieldBeDisabled}
+              required={i.required}
+              fullWidth
+              value={i.value.get_involved}
+              multiline
+              inputProps={{ maxLength: 250 }}
+              onChange={(event) =>
+                handleTextFieldChange("organization_size_and_involvement", event.target.value)
+              }
+            />
+            <div className={classes.marginBottom} />
+          </>
+        );
```

Comment:
This whole logic should be moved to a seperate organization-specific component as discussed. This will make it much simpler

Reply:
Logic removed.